# U.S. Medical Insurance Costs
This portfolio project has been done by RACHEL LEE.
### Introduction
This document shows the summary and analysis of the medical insurance costs with various demographics (age, sex, BMI, number of children, smoking status, and location). The analysis includes the observation of each demographic against medical insurance costs and examines the trends on the graphs. At the end, the document makes the prediction that which variable might be able to affect the results the most and suggests any potential bias that resides.
### Data and Demographics
This project used the provided csv file "insurance.csv" for analysis. The file has been downloaded from the following link.

https://www.kaggle.com/mirichoi0218/insurance

With the csv file downloaded, the data have been analyzed using python. First, the pandas module was imported to use the dataframe structure for easier data management.

In [3]:
import pandas as pd

Using the pandas module, the data in "insurance.csv" can be imported to the code.

In [9]:
pd.read_csv("insurance.csv")

age     sex     bmi  children smoker     region      charges
0      19  female  27.900         0    yes  southwest  16884.92400
1      18    male  33.770         1     no  southeast   1725.55230
2      28    male  33.000         3     no  southeast   4449.46200
3      33    male  22.705         0     no  northwest  21984.47061
4      32    male  28.880         0     no  northwest   3866.85520
...   ...     ...     ...       ...    ...        ...          ...
1333   50    male  30.970         3     no  northwest  10600.54830
1334   18  female  31.920         0     no  northeast   2205.98080
1335   18  female  36.850         0     no  southeast   1629.83350
1336   21  female  25.800         0     no  southwest   2007.94500
1337   61  female  29.070         0    yes  northwest  29141.36030

[1338 rows x 7 columns]

As in the output, the columns include the following variables: age, sex, bmi, children, smoker, region, and charges. The specs of the dataframe can be shown as below. The specs show that there are 1338 entries in total, and each variable has different data types. Age and children are integers, bmi and charges are float, and sex, smoker, and region are objects, which is strings. Therefore, different analysis method needs to be applied based on the data types.

In [12]:
df = pd.read_csv("insurance.csv")
print(df.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1338 entries, 0 to 1337
Data columns (total 7 columns):
 #   Column    Non-Null Count  Dtype  
---  ------    --------------  -----  
 0   age       1338 non-null   int64  
 1   sex       1338 non-null   object 
 2   bmi       1338 non-null   float64
 3   children  1338 non-null   int64  
 4   smoker    1338 non-null   object 
 5   region    1338 non-null   object 
 6   charges   1338 non-null   float64
dtypes: float64(2), int64(2), object(3)
memory usage: 73.3+ KB
None


Before we perform the analysis, we need to split the dataframe into individual variables to easily manage the data set.

In [14]:
age = df.age
sex = df.sex
bmi = df.bmi
child = df.children
smoker = df.smoker
region = df.region

#### Age
The average age of the entire entries is 30.

In [15]:
print(age.mean())

39.20702541106129
